examples/pngWithRoundedShapes.php

```diff
@@ -0,0 +1,210 @@
+<?php
+
+use chillerlan\QRCode\Common\EccLevel;
+use chillerlan\QRCode\Data\QRMatrix;
+use chillerlan\QRCode\Output\QRGdImage;
+use chillerlan\QRCode\Output\QROutputInterface;
+use chillerlan\QRCode\QRCode;
+use chillerlan\QRCode\QROptions;
+
+require_once __DIR__.'/../vendor/autoload.php';
+
+// --------------------
+// Class definition
+// --------------------
+
+class QRCustomPNG extends QRGdImage {
+
+    protected function setPixel(int $x, int $y, int $M_TYPE): void {
```

Comment:
The `setPixel()` method has ben renamed to `module()` in the current `dev-main`.